unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioSource.cs

```diff
@@ -71,7 +71,7 @@ private void ApplyCurrentModel()
                 return;
             }
 
-            audioSource.volume = (scene.sceneData.id == CommonScriptableObjects.sceneID.Get()) ? model.volume : 0f;
+            audioSource.volume = ((scene.sceneData.id == CommonScriptableObjects.sceneID.Get()) || (scene is GlobalScene && ((GlobalScene)scene).isPortableExperience)) ? model.volume : 0f;
```

Comment:
That new check can probably be expressed as `(scene as GlobalScene)?.isPortableExperience`, it's just a suggestion, skip it if you don't think it's better.

Reply:
I tried to use this expression as you suggest but it seems the OR operator don't like it =(

![image](https://user-images.githubusercontent.com/64659061/108338313-6799ba00-71d6-11eb-9775-8dad8041e34f.png)